packages/web-components/src/components/va-breadcrumbs/va-breadcrumbs-slot.css

```diff
@@ -64,3 +65,22 @@ va-breadcrumbs li > a[aria-current='page'] {
   pointer-events: none;
   text-decoration: none;
 }
+
+/* USWDS v3 styles */
+va-breadcrumbs .usa-breadcrumb__list-item {
+  display: inline-block;
+  line-height: 1.5;
+  margin: 4px 0;
+  padding: 4px 0;
+  vertical-align: middle;
+}
+
+va-breadcrumbs .usa-breadcrumb__list-item::after {
+  content: '\203A';
+  display: inline-block;
+  padding: 0 5.6px;
+}
+
+va-breadcrumbs .usa-breadcrumb__list-item:last-child::after {
+  content: '';
+}
```

Comment:
I don't understand why we are adding these USWDS styles here unless we have a VA need to override something. 

```
va-breadcrumbs .usa-breadcrumb__list-item {
  display: inline-block;
  line-height: 1.5;
  margin: 4px 0;
  padding: 4px 0;
  vertical-align: middle;
}
```

In this part, are we intentionally overriding the icon in order to use Font Awesome?

```
va-breadcrumbs .usa-breadcrumb__list-item::after {
  content: '\203A';
  display: inline-block;
  padding: 0 5.6px;
}
```

The reason I'm pointing this out is that we want to pull styles directly from the USWDS modules so that we get their updates automatically. Any time we add/override USWDS styles in the component itself, we lose that so I want to make sure there is a specific reason for doing it.

Reply:
Well, I reason I added it because it shows as a list without icon.

Reply:
Is there a Sketch file for this component by chance?

When I look at the USWDS example, there is an icon they provide as part of the breadcrumb module. Sometimes we are using our own Fontawesome icons though so your addition would make sense if that's what the Sketch file shows. But if we are using the USWDS icon, we should make sure that works:

![Screenshot 2023-06-06 at 11 48 02 AM](https://github.com/department-of-veterans-affairs/component-library/assets/872479/3b4af77d-629f-4faf-9ff9-c569d4eb7552)